kfs-core/src/main/java/edu/arizona/kfs/coa/businessobject/AccountExtension.java

```diff
@@ -32,6 +32,7 @@
     // Helper Objects
     private transient volatile BudgetShellCode budgetShell;
     private transient volatile CrossOrganizationCode crossOrganization;
+    private FACostSubCategory faCostSubCategory;    // FA Subcategory Object

```

Comment:
Please change to:
private transient volatile FACostSubCategory faCostSubCategory;